src/config.tsx

```diff
@@ -1,5 +1,5 @@
 export const BACKEND_URL = process.env.REACT_APP_BACKEND_URL || "";
-export const ENABLE_WEBSOCKETS_SUPPORT = false;
+export const ENABLE_WEBSOCKETS_SUPPORT = process.env.REACT_APP_ENABLE_WEBSOCKETS_SUPPORT === "true" || false;
 export const BACKEND_WS_URL = process.env.REACT_APP_BACKEND_WS_URL || "";
-export const USE_SECURE_COOKIE = process.env.REACT_APP_USE_SECURE_COOKIE === "false" || true;
+export const USE_SECURE_COOKIE = process.env.REACT_APP_USE_SECURE_COOKIE !== "false";
 export const DEBUG = process.env.REACT_APP_DEBUG === "true" || false;
```

Comment:
In prod, this entire file is replaced. We still don't have a good solution for this. A matter for another PR.

Reply:
Indeed, and I'm not sure we *can* have a better solution, since we only deploy static files in production (though I'm sure some crazy people run Node server directly in production and can use the environment for config).

I've mentioned this bit in #180